muller/dataio/genotypecollection.py

```diff
@@ -59,9 +59,9 @@ def get_genotype_from_annotation(self, value: str) -> List[Genotype]:
 			all_annotations = [i.annotations for i in self.values()]
 			available = sorted(itertools.chain.from_iterable(all_annotations))
 			try:
-				from fuzzywuzzy import process
-				similar = process.extract(value, available)
-				similar = [i[0] for i in similar if i[1] > 80]
+				from rapidfuzz import process
+				similar = process.extract(value, available, score_cutoff=80)
+				similar = [i[0] for i in similar]
```

Comment:
@cdeitrick you might want to use the `score_cutoff` parameter since it allows the extract function to exit early when it e.g. knows a comparision can not reach this score based on string lengths